src/Compilers/CSharp/Portable/xlf/CSharpResources.pt-BR.xlf

```diff
@@ -4387,7 +4387,7 @@ Um bloco catch() depois de um bloco catch (System.Exception e) poderá capturar
       </trans-unit>
       <trans-unit id="ERR_AnonDelegateCantUse">
         <source>Cannot use ref, out, or in parameter '{0}' inside an anonymous method, lambda expression, query expression, or local function</source>
-        <target state="needs-review-translation">Não é possível usar parâmetro ref ou out "{0}" dentro de um método anônimo, expressão lambda ou expressão de consulta</target>
+        <target state="translated">Não é possível usar parâmetro ref, out, ou in "{0}" dentro de um método anônimo, expressão lambda, expressão de consulta, ou função local</target>
```

Comment:
Should this remain marked as "needs-review-translation"? I am not sure how will this change flow into our internal LOC database, if one still exists. CC @tmeschter

Reply:
let me know which state that should be and I can make the change.